src/app/shared/components/user-info/user-info.component.html

```diff
@@ -1,2 +1,9 @@
-<div>User login </div>
-<!-- <a mat-button color="primary">Logout</a> -->
+<div *ngIf="user">
+  <span class="info">Welcome {{user.email}}!
+  <a routerLink="/logout">Logout</a></span>
+</div>
+
+<div *ngIf="!user">
+    <a routerLink="/login">Login</a>
```

Comment:
It seems like you don't have appropriate route in your modules ( was redirected on 404 )
But this issue we'll solve in the future task :)